components/Eingabefeld 1/index.js

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+import { locations } from "../../db/data.js";
+import moment from "moment";
+
+const now = moment();
+const lockedTime = now.set().format("HH:mm");
+
+export default function InformationPage() {
+  return (
+    <OverAllWrapper>
+      <DisplayFlex>
+        Kleidung <br /> hinterlegt
+        <DarkGreenContainer>
+          <p>
+            <strong>{lockedTime} Uhr</strong>
+          </p>
+        </DarkGreenContainer>
+      </DisplayFlex>
+      <DisplayFlex>
+        {locations[0].name}
+        <br />
```

Comment:
Use of `<br />` It is not recommended to use `<br />` for layout purposes, as it is more difficult to interpret for screen readers. CSS layout techniques such as flexbox or grid should be used instead.